services/gateway/pkg/config/config.go

```diff
@@ -76,9 +76,9 @@ type GRPCConfig struct {
 }
 
 type StorageRegistry struct {
-	Driver              string   `yaml:"driver"` // TODO: configure via env?
-	Rules               []string `yaml:"rules"`  // TODO: configure via env?
-	JSON                string   `yaml:"json"`   // TODO: configure via env?
+	Driver              string   `yaml:"driver" env:"GATEWAY_STORAGE_REGISTRY_DRIVER" desc:"The driver name of the storage registry to use."`
+	Rules               []string `yaml:"rules" env:"GATEWAY_STORAGE_REGISTRY_RULES" desc:"The rules for the storage registry."`
+	JSON                string   `yaml:"json" env:"GATEWAY_STORAGE_REGISTRY_CONFIG_JSON" desc:"Additional configuration for the storage registry in json format."`
```

Comment:
Q: how does the json get assigned to the variable? Imho a reference to a file would do this.
See: [USERS_LDAP_CACERT](https://doc.owncloud.com/ocis/next/deployment/services/s-list/users.html#configuration) where we have a default file name and default path/file if not otherwise defined: `If not defined, the root directory derives from $OCIS_BASE_DATA_PATH:/idm.` Just my 2c.